Source/Core/Common/FileUtil.cpp

```diff
@@ -755,8 +772,10 @@ std::string GetSysDirectory()
 	sysDir = GetBundleDirectory() + DIR_SEP + SYSDATA_DIR;
 #elif defined (_WIN32)
 	sysDir = GetExeDirectory() + DIR_SEP + SYSDATA_DIR;
+#elif defined (LINUX_LOCAL_DEV)
+	sysDir = GetExeDirectory() + DIR_SEP + "Sys";
```

Comment:
I tried that but the path ended up to be /path/to/dolphin//usr/local/dolphin/sys